docker-compose.yml

```diff
@@ -0,0 +1,26 @@
+version: "3.7"
+services:
+  postgres:
+    image: "postgres:12.5"
+    container_name: "cloud-postgres"
+    restart: on-failure
+    networks:
+    - cloud
+    ports:
+    - "5430:5432"
```

Comment:
I think Gabe uses 5433 here, but that's something that is taken by CWS for me so I've gone for 5430 instead. 0/5 on what we want here.

Reply:
Yep, that totally works. 👍